config/templates/metasploit-framework-wrappers/msfupdate.erb

```diff
@@ -100,6 +100,16 @@ EOF
   yum install -y metasploit-framework
 }
 
+install_pkg()
+{
+  (
+    cd ~/Downloads
```

Comment:
It does, but the user owns ```~/Downloads```, so they can ```rm -f``` it. I had ```rm -fv``` in this function, but I'm leaving it off for now. Please don't merge yet. I'll add it once I add status messages.